erpnext_germany/erpnext_germany/doctype/vat_id_check/vat_id_check.json

```
@@ -0,0 +1,260 @@
+{
+ "actions": [],
+ "creation": "2023-03-26 18:28:14.770174",
+ "default_view": "List",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "customer",
+  "customer_vat_id",
+  "trader_name",
+  "customer_address",
+  "trader_street",
+  "trader_postcode",
+  "trader_city",
+  "column_break_hmgxr",
+  "status",
+  "section_break_xljfy",
+  "company",
+  "column_break_scjlu",
+  "requester_vat_id",
+  "section_break_dowxn",
+  "is_valid",
+  "trader_name_match",
+  "trader_street_match",
+  "trader_postcode_match",
+  "trader_city_match",
+  "column_break_c4uco",
+  "request_id",
+  "actual_trader_name",
+  "actual_trader_address"
+ ],
+ "fields": [
+  {
+   "fieldname": "customer",
+   "fieldtype": "Link",
+   "in_standard_filter": 1,
+   "label": "Customer",
+   "options": "Customer",
+   "set_only_once": 1
+  },
+  {
+   "fieldname": "column_break_hmgxr",
+   "fieldtype": "Column Break"
+  },
+  {
+   "fieldname": "request_id",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "Request ID",
+   "no_copy": 1,
+   "read_only": 1
+  },
+  {
+   "default": "0",
+   "fieldname": "is_valid",
+   "fieldtype": "Check",
+   "label": "Is Valid",
+   "no_copy": 1,
+   "read_only": 1
+  },
+  {
+   "fieldname": "company",
+   "fieldtype": "Link",
+   "label": "Company",
+   "options": "Company",
+   "set_only_once": 1
+  },
+  {
+   "fieldname": "section_break_xljfy",
+   "fieldtype": "Section Break",
+   "label": "Requester"
+  },
+  {
+   "fieldname": "column_break_scjlu",
+   "fieldtype": "Column Break"
+  },
+  {
+   "depends_on": "eval:!doc.__islocal",
+   "fieldname": "section_break_dowxn",
+   "fieldtype": "Section Break",
+   "label": "Result"
+  },
+  {
+   "fieldname": "column_break_c4uco",
+   "fieldtype": "Column Break"
+  },
+  {
+   "fetch_from": "customer.tax_id",
+   "fetch_if_empty": 1,
+   "fieldname": "customer_vat_id",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "in_standard_filter": 1,
+   "label": "Customer VAT ID",
+   "reqd": 1,
+   "set_only_once": 1
+  },
+  {
+   "fetch_from": "company.tax_id",
+   "fetch_if_empty": 1,
+   "fieldname": "requester_vat_id",
+   "fieldtype": "Data",
+   "label": "Requester VAT ID",
+   "set_only_once": 1
+  },
+  {
+   "default": "Planned",
+   "fieldname": "status",
+   "fieldtype": "Select",
+   "label": "Status",
+   "options": "Planned\nRunning\nCompleted\nUnavailable\nInvalid Input",
```

```suggestion
   "options": "Planned\nRunning\nCompleted\nService Unavailable\nInvalid Input",
```
As per what is used everywhere else